README.md

```diff
@@ -6,23 +6,37 @@ likely to be accepted.
 
 ## This Site
 
-### Compiling
+### Installing
 
 This site is mostly compiled by magic (specifically
 [Jekyll](http://jekyllrb.com/) and [Sass](http://sass-lang.com/) which will
-have to be installed before you can do anything. The following commands will
-install the necessary software:
+have to be installed before you can do anything.
+
+Unless you have a good reason not to, you will probably want to use Bundler
+to install the ruby dependencies. Just run this from the root directory of
+this repo:
+
+    bundle install
+
+Also, make sure to check out the latest version of the `theme` submodule:
+
+    git submodule init
```

Comment:
This could be `git submodule update --init` (which works first time and subsequent times, so below can then be removed).